.github/workflows/archived/production-deployment.yml

```
@@ -212,7 +212,7 @@ jobs:
       - name: Database Integration Tests
         env:
           ConnectionStrings__DefaultConnection: "Host=localhost;Port=5432;Database=terrafusion_test;Username=test_user;Password=test_password"
-          ConnectionStrings__Redis: "localhost:6379"
+          ConnectionStrings__Redis: "localhost:\${{TF_REDIS_PORT:-6379}}"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 215: found unknown escape character

(syntax-check)